microprofile.jdt/org.eclipse.lsp4mp.jdt.core/src/main/java/org/eclipse/lsp4mp/jdt/core/PropertiesManager.java

```diff
@@ -386,11 +386,11 @@ private SearchPattern createSearchPattern() {
 	 */
 	private static IJavaSearchScope createJavaSearchScope(IJavaProject fakeProject, boolean excludeTestCode,
 			IJavaElement[] elements, int includeMask) {
-		HashSet projectsToBeAdded = new HashSet(2);
+		HashSet<JavaProject> projectsToBeAdded = new HashSet<JavaProject>(2);

```

Comment:
Set<JavaProject>

Reply:
Unfortunately, it looks like a JDT API requires it to be a HashSet, although I'll update it to use the diamond operator.